src/main/groovy/edu/oregonstate/mist/tdwrapper/db/TDServicesStaticJsonDAO.groovy

```diff
@@ -0,0 +1,66 @@
+package edu.oregonstate.mist.tdwrapper.db
+
+import edu.oregonstate.mist.api.jsonapi.ResourceObject
+import groovy.json.JsonSlurper
+
+class TDServicesStaticJsonDAO {
+
+    private File servicesFile
+
+    TDServicesStaticJsonDAO(String sourceJsonFile) {
+        servicesFile = new File(sourceJsonFile)
+    }
+
+    /**
+     * Get all services
+     * @return
+     */
+    public List<ResourceObject> getTDServices() {
+        def servicesRaw = getJsonRaw()
+
+        List<ResourceObject> services = []
+
+        servicesRaw.each { id, service ->
+            services.add(getResourceObject(service))
+        }
+
+        services
+    }
+
+    /**
+     * Get service by ID
+     * @param id
+     * @return
+     */
+    public ResourceObject getTDServiceByID(String id) {
+        def servicesRaw = getJsonRaw()
+
+        if (!servicesRaw[id]) {
+            return null
+        }
+
+        getResourceObject(servicesRaw[id])
+    }
+
+    /**
+     * Helper method to create resource object
+     * @param serviceRaw
+     * @return
+     */
+    private ResourceObject getResourceObject(def serviceRaw) {
+        new ResourceObject(
+                id: serviceRaw['ID'],
+                type: "service",
+                attributes: serviceRaw
+        )
+    }
+
+    /**
+     * Helper method to slurp json file into a def
+     * @return
+     */
+    private def getJsonRaw() {
+        JsonSlurper jsonSlurper = new JsonSlurper()
+        jsonSlurper.parseText(servicesFile.getText())
```

Comment:
why JsonSlurper instead of jackson? just curious

Reply:
I don't really have a good reason. I didn't need to map the json into some sort of POGO. Basically just needed something to read json from a file without doing much data manipulation.